app-frontend/src/app/services/tools/tool.service.js

```diff
@@ -42,6 +43,55 @@ export default (app) => {
             return this.Tool.query(params).$promise;
         }
 
+        searchQuery() {
+            let deferred = this.$q.defer();
+            let pageSize = 1000;
+            let firstPageParams = {
+                pageSize: pageSize,
+                page: 0,
+                sort: 'createdAt,desc'
```

Comment:
This sort order doesn't seem to be taking effect (which may be a backend bug): if I inspect the responses from the API to the search queries, they aren't ordered by `createdAt` (or anything else, as far as I can tell). There might need to be an issue opened for this.